build.sbt

```diff
@@ -279,7 +279,8 @@ def scalaVersionSince = Map(
 
 def since10 = Seq("1.0.0") ++ since11
 def since11 = Seq("1.1.0") ++ since12
-def since12 = Seq("1.2.2")
+def since12 = Seq("1.2.2") ++ since13
+def since13 = Seq("1.3.0")
```

Comment:
As @ignasi35 mentioned in #1082, maybe it's better to use 1.3.10 here.

Reply:
I suggested `1.3.10` because #1082, but maybe we can find an earlier version (not as old as `1.3.0`) that provides a good trade-off. The further back we go, the more exclusions we have to add (see below the list of exclusions required using `1.3.10`).


```
PF.exclude[DirectMissingMethodProblem]("c.l.l.s.testkit.TopicStub#SubscriberStub.this")
PF.exclude[IncompatibleMethTypeProblem]("c.l.l.s.persistence.cassandra.testkit.TestUtil#AwaitPersistenceInit.persist")
PF.exclude[IncompatibleMethTypeProblem]("c.l.l.s.persistence.cassandra.testkit.TestUtil#AwaitPersistenceInit.persistAsync")
PF.exclude[IncompatibleMethTypeProblem]("c.l.l.s.persistence.testkit.AbstractTestUtil#AwaitPersistenceInit.persist")
PF.exclude[IncompatibleMethTypeProblem]("c.l.l.s.persistence.testkit.AbstractTestUtil#AwaitPersistenceInit.persistAsync")
PF.exclude[IncompatibleResultTypeProblem]("c.l.l.s.api.AdditionalConfiguration.configuration")
PF.exclude[IncompatibleResultTypeProblem]("c.l.l.s.playjson.JsonMigration.transform")
PF.exclude[InheritedNewAbstractMethodProblem]("c.l.l.s.api.LagomConfigComponent.config")
PF.exclude[ReversedMissingMethodProblem]("c.l.l.s.api.ServiceLocator.locateAll")
PF.exclude[ReversedMissingMethodProblem]("c.l.l.s.api.broker.Subscriber.withMetadata")
PF.exclude[ReversedMissingMethodProblem]("c.l.l.s.client.CircuitBreakerComponents.circuitBreakersPanel")
```

Reply:
I think 1.3.3 is a good trade-off, it will have around half of the exclusions (~20)